R/authors-at-r.R

```diff
@@ -47,12 +47,54 @@ set_author_field <- function(authors, which, field, value) {
 
 
 ensure_authors_at_r <- function(obj) {
+  coerce_authors_at_r(obj)
   if (! obj$has_fields("Authors@R")) {
     stop("No 'Authors@R' field!\n",
          "You can create one with $add_author")
   }
 }
 
+coerce_authors_at_r <- function(obj) {
+  has_authors_at_r = obj$has_fields("Authors@R")
+  has_author = obj$has_fields("Author")
+  if (! (has_authors_at_r | has_author) ) {
+    stop("No 'Authors@R' or 'Author' field!\n",
+         "You can create one with $add_author")
+  }
+  if (has_author & has_authors_at_r) {
+    # Delete Author as it has Authors@R
+    obj$del("Author")
```

Comment:
You cannot modify `obj`, it will change (the representation of) the `DESCRIPTION` file.